### ISSUE TITLE
Disable "Order Generation" when already done

### ISSUE THREAD

**Issue by kousu** [savoirfairelinux/sous-chef#797]
### Expected Behaviour

On /delivery/order/, I would expect that once the order generation is done, the button to do it would be disabled or at least greyed out.

### Actual Behaviour

![orders_generate](https://user-images.githubusercontent.com/987487/29340214-d8bb65e6-81eb-11e7-8df7-c6cfcd7c480c.png)

 It took me 20 minutes wondering why the button was spinning everytime I clicked it and not doing anything before I realized it had loaded with the orders pre-generated.

### Steps to reproduce

Click "Kitchen Count"

**Comment by lingxiaoyang** [comment]
@kousu Perhaps it is better to explain the use of this button somewhere on the page. I was a little bit confused at first but I did find the button useful in particular situations (in case the order generation doesn't work, for example).

@lamontfr Do you know more of the design of this button? I have no prior knowledge on this part...

**Comment by lamontfr** [comment]
@kousu @lingxiaoyang This button is necessary to handle these cases : 
- The customer manager creates new ongoing clients (or changes the delivery mode to ongoing for existing episodic clients) after the scheduled order generation has run. The button will then generate the day's orders for such ongoing clients.
- The customer manager changes the status to active for some clients after the scheduled order generation has run. The button will then generate the day's orders for such ongoing clients.

Therefore it is good practice to click "generate orders" as the first step of the kitchen count.
